docs/user-docs/latest/settings/configuring-users-permissions-plugin-settings.md

```diff
@@ -1,10 +1,10 @@
-# Configuring Users & Permissions plugin
+# Configuring Users & Permissions plugin settings
 
-The Users & Permissions plugin is managed from the *Users & Permissions plugin* settings section, accessible from *General > Settings* in the main navigation of the admin panel. This settings section allows to configure the available providers and email templates, and the advanced settings of the plugin. It also allows to define the end-users roles and their related permissions (see [Configuring end-users roles](../users-roles-permissions/configuring-end-users-roles.md)).
+The Users & Permissions plugin is managed from the *Users & Permissions plugin* settings section, accessible from *General > Settings* in the main navigation of the admin panel. This settings section allows to configure the available providers, email templates and the advanced settings of the plugin. It also allows to define the end-users roles and their related permissions (see [Configuring end-users roles](../users-roles-permissions/configuring-end-users-roles.md)).
 
 ## Configuring providers
 
-The Users & Permissions plugin allows to enable and configure providers, for end-users to login via a third-party provider to access the content of a Strapi application. By default, a list of 12 providers is available including one, "Email", enabled by default for all Strapi applications with the Users & Permissions plugin installed.
+The Users & Permissions plugin allows to enable and configure providers, for end-users to login via a third-party provider to access the content of a font-end application through the API. By default, a list of 12 providers is available including one, "Email", enabled by default for all Strapi applications with the Users & Permissions plugin installed.
```

Comment:
This sentence troubles me.

It sounds like the API mentioned at the end is the one from the front-end application.
But it's not, we are talking about the Strapi application API.

The end-user will go in the front-end application login page.
Click on the Google button, will login thanks to this Strapi provider feature.
The user is now login into the front-end application.
And this front-end application will call the Strapi application API (with the token from the end-user) to get and display the data.

Reply:
Not sure if I am writing in the right place but I think we should not mention the number of providers we have in the default list (actually I think there are already 15). The rest of the sentence is clear enough and it could avoid updates if we happen to add some default providers in the future.